src/MigrationTools.Clients.AzureDevops.ObjectModel/Processors/PipelineType.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace MigrationTools.Processors
+{
+
+    public enum PipelineType
+    {
+        build,
```

Comment:
They have to be lowercase since I use them to build the request URL
`string baseUrl = $"{Target.Organisation}/{Target.Project}/_apis/{pipelineType}/definitions?api-version=5.1-preview";`

Reply:
I am not sure if the httpclient cares about casing, pretty sure http stack is case insensitive on url?